packages/ui-components/src/NetworkManger/NetworkManager.tsx

```diff
@@ -30,6 +30,8 @@ import { Pallet } from '@webb-dapp/ui-components/styling/colors';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import styled, { css } from 'styled-components';
 
+import { above, useBreakpoint } from '../utils/responsive-utils';
```

Comment:
consider using the paths aliasing import
Ex `import {above,useBreakpoing} from '@webb-dapp/ui-components/utils'`